readme.txt

```diff
@@ -8,45 +8,60 @@ Stable tag: 2.0.0
+3. Search for Acquired.com and click Install Now.
+4. Activate the plugin and navigate to WooCommerce → Settings → Payments.
+5. Enter your Acquired.com credentials and enable your preferred payment methods — Cards, Apple Pay, Google Pay, or Pay by Bank.
 
-- **Pay by Card:** Securely accept Visa, Mastercard & American Express using EMV 3-D Secure (version 2.2)
-- **Digital Wallets:** Add Apple Pay & Google Pay to the Product, Cart or Checkout pages of your store to deliver a seamless checkout experience
-- **Customisable Checkout:** Match your brand with full control to style the payment form to meet your requirements
-- **Saved Cards:** Enable faster checkout for returning customers by letting them reuse previously stored cards
-- **Manage your payments:** Update the payment statuses from within WooCommerce
-- **Always available:** Excellent customer service and reliability (99.999% uptime) at our core
-- **PCI DSS:** Remain fully compliant with the lowest amount of effort (SAQ-A) using our hosted solution
+You’re now ready to start accepting payments with Acquired.com.
 
-== Quick & Easy Setup ==
+For detailed setup instructions, visit our [WooCommerce integration guide](https://docs.acquired.com/docs/woocommerce-v2).
 
-Please [contact us](https://acquired.com/contact) to get a test account so you can try out the extension.
+== Frequently Asked Questions ==
+= Do I need an Acquired.com account to use this plugin? =
+Yes. You’ll need an Acquired.com merchant account to process payments. Contact us to get started.
 
-1. Access your WordPress dashboard.
-2. Select **Plugins** from the left navigation menu followed by **Add New**
-3. Search for **Acquired.com** using the top right-hand corner search box
-4. Select **Install Now**
-5. Once Installed you can enable and configure all things payments in the WooCommerce settings
-6. Configure a few basic details and you are up and running
+= Is Acquired.com PCI DSS compliant? =
+Yes. Acquired.com is PCI DSS Level 1 compliant. Using this plugin via Hosted Checkout keeps you within SAQ-A scope.
 
-Read our documentation [here](https://docs.acquired.com/docs/woocommerce-1) and our support teams are on hand to help with any questions you might have so please [contact us](https://acquired.com/contact).
+= Can I use Apple Pay or Google Pay with this plugin? =
+Yes. Both Apple Pay and Google Pay are available via the Hosted Checkout integration.
 
-== Screenshots ==
+= Does this plugin support recurring payments or WooCommerce Subscriptions? =
+Not at this time. The plugin currently supports one-off and stored card payments only.
 
-1. Configuration
-2. Card Payment
-3. Apple Pay
-4. Google Pay
+= Where can I get support? =
+Our UK-based support team is available to help you. [Contact support](https://www.acquired.com/contact/) or visit our [Developer Documentation](https://docs.acquired.com/docs/woocommerce-v2).
 
 == Changelog ==
 
-= 2.0.0 =
+= 2.0.0 - 2025/10/25 =
```

Comment:
Should be `2025/10/23` I think?

Reply:
Hi @SlobodanAlavanjaNeuralab 

Rich mentions [here](https://trello.com/c/r5obKRwd/80-plugin-release-github-action-request#comment-68f2511d0f570ed9c4c8d584) that the release will be the 22nd?